cfme/control/explorer.py

```diff
@@ -1429,6 +1430,8 @@ def _fill(self, action):
         of forms."""
         fill(self.form, dict(description=self.description, action_type=self.action_type))
         if self.sub_forms[self.action_type] is not None:
+            # Waiting for tree loading
+            sleep(2)
```

Comment:
Can you provide an explanation? We try to avoid sleep everywhere where possible. Possibly you can wait_for_element?